src/data_generators/tests/heavy_tailed_impression_generator_test.py

```diff
@@ -23,10 +22,10 @@
 
 class ImpressionGeneratorTest(absltest.TestCase):
     def test_heterogeneous_impression_generator(self):
-        generator = HeavyTailedImpressionGenerator(5, 2, RandomState(0))
+        generator = HeavyTailedImpressionGenerator(5, 2)
         generated_ids = generator()
         self.assertEqual(set(generated_ids), set([0, 1, 2, 3, 4]))
-        self.assertLen(generated_ids, 33)
+        self.assertLen(generated_ids, 13)
```

Comment:
How do we get the number `13` here? Is it solely based on the result of seed==1?

Reply:
Yes. Result changes when using a different seed.

(I wrote this test before and now realized that a test based on random seed is not a good practice. Let's keep it for now and I'll use "fake seed" for test in the future.)